packages/cashier/src/stores/general-store.ts

```diff
@@ -164,17 +165,17 @@ export default class GeneralStore extends BaseStore {
         const { account_transfer } = modules.cashier;
 
         if (this.active_container === account_transfer.container) {
-            this.percentage = +((amount / +account_transfer.selected_from.balance) * 100).toFixed(0);
+            this.percentage = (amount / Number(account_transfer.selected_from.balance)) * 100;
         } else {
-            this.percentage = +((amount / +client.balance) * 100).toFixed(0);
```

Comment:
Because `this.percentage` is a number and `toFixed(0)` makes it string.

Reply:
@hamid-deriv In my opinion, would be safer to keep it and just cast it to number afterward because I'm not sure about the logic and it might break something as the results are different. What do you say? 🤔 
<img width="298" alt="image" src="https://user-images.githubusercontent.com/72082844/192964575-b7589760-764a-4d62-bbe6-57666dc7cce8.png">

Reply:
You're right. Fixed.